packages/scrapers-v2/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const COURSE_REGEX = /([A-Z]{2,4})\s([0-9]{4})/g;
```

Comment:
add comment at top of file explaining what it is for

Reply:
updated!